README.md

```diff
@@ -52,6 +52,12 @@ _Example:_ `name: GitHub scan ${{ github.sha }}`
 
 _Example:_ `api_token: ${{ secrets.NEURALEGION_TOKEN }}`
 
+### `project_id`
+
+Provide project-id if you want to run a scan for a specific project. If you don't provide project-id, scan will run under Default project.
```

Comment:
@bramkor - see that here you use an hyphen and not an underscore - like the param name.